### PR TITLE
Added panther dependency

### PR DESCRIPTION
**Pull request by mickaelandrieu**
`make install && make phpunit` should works as expected.

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -21,3 +21,7 @@ DATABASE_URL=sqlite:///%kernel.project_dir%/var/data/blog.sqlite
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
 ###< symfony/swiftmailer-bundle ###
+
+# Configuration for Panther using Docker
+PANTHER_NO_SANDBOX=1
+PANTHER_WEB_SERVER_PORT=9001
```
Comment (mickaelandrieu) [inline .env.dist]: missing EOL

File: tests/Smoke/BlogSmokeTest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *     $ ./vendor/bin/simple-phpunit
+ */
+class BlogSmokeTest extends PantherTestCase
+{
+    public function testHomePageIsAvailable()
+    {
+        $client = static::createPantherClient();
+
+        $client->request('GET', '/');
+        $response = $client->getResponse();
```
Comment (mickaelandrieu) [inline tests/Smoke/BlogSmokeTest.php]: note to myself, $response is not available in Panther => to be reverted.

Panther is not adapted for this kind of test.

File: .env.dist (hunk 1 above)
Comment (dunglas) [reply .env.dist]: It would be weird that it works, it should be set in `phpunit.xml`.

**Comment by mickaelandrieu** [comment]
Thanks for the hint about `unzip` @dunglas! Tests are failing but it's totally unrelated to panther.

As this project is mainly a fork of Symfony Demo: do you think it's worth to suggest to add some Panther use/tests in the former repository?

I don't have too much time these days, so I need to be sure this won't be rejected if I invest time :angel: 

Mickaël

File: tests/Smoke/BlogSmokeTest.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * Le Smoke testing permet d'identifier si on
```
Comment (mickaelandrieu) [inline tests/Smoke/BlogSmokeTest.php]: need to be done in english

**Comment by dunglas** [comment]
> As this project is mainly a fork of Symfony Demo: do you think it's worth to suggest to add some Panther use/tests in the former repository?

Definitely!

**Pull request merged**